src/library/scala/concurrent/ExecutionContext.scala

```diff
@@ -54,8 +54,17 @@ import scala.annotation.implicitNotFound
  * Application callback execution can be configured separately.
  */
 @implicitNotFound("""Cannot find an implicit ExecutionContext. You might pass
-an (implicit ec: ExecutionContext) parameter to your method
-or import scala.concurrent.ExecutionContext.Implicits.global.""")
+an (implicit ec: ExecutionContext) parameter to your method.
```

Comment:
Perhaps something like this,

`an (implicit ec: ExecutionContext) parameter: to your method, to the constructor, or obtain it via other means.`